### ISSUE TITLE
There is no obvious way to import an external object to Editor

### ISSUE THREAD

**Issue by carlfravel** [decentraland/editor-legacy#9]
Summary: what are the recommended practices for importing foreign objects into IPFS or other web storage and then getting them into the "inline" storage (in the A-Frame sense of "inline")?

This MIGHT only be a documentation request.

Steps to reproduce the UX problem:
1. Open editor, new or existing scene
2. In the DCL Editor scenegraph panel on left, click on Add... - .obj
// see a selection tool appear at the origin
// optionally move this to somewhere you want the object to initially appear.
3. Scroll to bottom of the Components panel on right
// observe a place to enter paths for .mtl and .obj
// If you look at the paths for the DCL logo disk object, you see paths that are relative to some root which is nowhwere documented what it is.
// /obj/Decentral_2.mtl
// /obj/Decentral_2.obj
4. Wonder how to get objects and materials uploaded in a way that they can then be used with paths like this
5. Tried putting the .obj and .mtl into the A-Frame Asset Uploader (UploadCare) https://cdn.aframe.io/
6. Tried putting the full paths with domain names of the uploaded files into the scene.
7. The object appeared (the mtl didn't, but that may be a different issue)

**Comment by michaltakac** [comment]
Right now @bnolan is working on object uploading features (.obj+.mtl) and I think it should be finalized in few days. We'll be using IPFS for storing textures. His progress so far: https://github.com/decentraland/editor/tree/feature/image-upload

**Comment by carlfravel** [comment]
FYI, 
I was able to get external object to at least partially work, if I uploaded it to a content delivery network, like cdn.aframe.io, and then using the full path to the uploaded URL in the DCL Editor's component panel at the bottom. The Object did appear in the scene. The material didn't, but that might be a problem with my material. But getting the content into IPFS and then having an appropriate scene-relative or land-relative or owner-relative path to use in the Editor is unclear or not yet supported.
